### PR TITLE
Fixes #26415 - debug export ordered now

### PR DESCRIPTION
**Pull request by lzap**
Untested, please test.

### CODE REVIEW

**Comment by lzap** [comment]
Can you guys ping me on IRC / mail once this gets merged of if there is problem? I have a BZ for this one.

File: extra/dynflow-debug.sh [hunk 1]
```diff
@@ -31,10 +31,10 @@ add_files /var/log/foreman/dynflow_executor*.output*
...
-    export_csv "select dynflow_actions.* from foreman_tasks_tasks join dynflow_actions on (foreman_tasks_tasks.external_id = dynflow_actions.execution_plan_uuid) where foreman_tasks_tasks.started_at > 'now'::timestamp - '${DYNFLOW_EXPORT_MONTHS:-1} months'::interval limit ${DYNFLOW_EXPORT_LIMIT:-100000}" "$DIR/dynflow_actions.csv"
-    export_csv "select dynflow_steps.* from foreman_tasks_tasks join dynflow_steps on (foreman_tasks_tasks.external_id = dynflow_steps.execution_plan_uuid) where foreman_tasks_tasks.started_at > 'now'::timestamp - '${DYNFLOW_EXPORT_MONTHS:-1} months'::interval limit ${DYNFLOW_EXPORT_LIMIT:-100000}" "$DIR/dynflow_steps.csv"
+    export_csv "select dynflow_execution_plans.* from foreman_tasks_tasks join dynflow_execution_plans on (foreman_tasks_tasks.external_id = dynflow_execution_plans.uuid) where foreman_tasks_tasks.started_at > 'now'::timestamp - '${DYNFLOW_EXPORT_MONTHS:-1} months'::interval order by foreman_tasks_tasks.started_at limit ${DYNFLOW_EXPORT_LIMIT:-100000}" "$DIR/dynflow_execution_plans.csv"
```
Comment (iNecas) [inline extra/dynflow-debug.sh]: aren't you missing `desc` there?

Comment (lzap) [reply extra/dynflow-debug.sh]: Is this a question? ;) Added "desc" for all order-bys. Rebased.

**Comment by lzap** [comment]
Rebased again, hello! :-)

**Pull request merged**